README.md

```diff
@@ -14,13 +18,37 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### [Node.js](http://nodejs.org)
+
+You may want to use [nvm](https://github.com/creationix/nvm) as your maanger of Node versions on your
+local machine.  Think RVM for Node.js
+
+You can install from [homebrew](http://brew.sh/) if you have that installed you should type:
```

Comment:
This sentence is a bit confusing. Maybe: `You can install nvm using homebrew. If you have homebrew installed, run:`. Maybe we should even include instructions for checking if they have homebrew installed? Might not be necessary.